cogboard-webapp/src/components/helpers/index.js

```diff
@@ -2,4 +2,50 @@ export const setSize = factor => ({ theme }) => `${theme.spacing(factor)}px`;
 
 export const splitPropsGroupName = (propName) => {
   return propName.includes('.') ? propName.split('.') : [undefined, propName];
-};
\ No newline at end of file
+};
+
+/**
+ * Sort object properties (only own properties will be sorted).
+ * @param {object} obj object to sort properties
+ * @param {string|int} sortedBy 1 - sort object properties by specific value.
+ * @param {bool} reverse false - reverse sorting.
+ * @returns {Array} array of items in [[key,value],[key,value],...] format.
+ */
+const sortObjectProperties = (obj, sortedBy, reverse) => {
```

Comment:
Is this really have to be that complicated? Why not just use comparator?
Something like:
```
function compare( a, b ) {
  if ( a.label< b.label){
    return -1;
  }
  if (a.label > b.label){
    return 1;
  }
  return 0;
}

endpoints.sort( compare );
```